apps/smime.c

```diff
@@ -117,7 +120,7 @@ const OPTIONS smime_options[] = {
     {"no-CAstore", OPT_NOCASTORE, '-',
      "Do not load certificates from the default certificates store"},
     {"nochain", OPT_NOCHAIN, '-',
-     "set PKCS7_NOCHAIN so certificates contained in the message are not used as untrusted CAs" },
+     "Do not use certs contained in the message as untrusted CAs for chain building" },
```

Comment:
This suggestion does not work because there is also `-nointern`, which can preclude the use of certs contained in the message as candidate signer certs.